tests/integration/index.ts

```diff
@@ -186,81 +214,79 @@ describe("Positive Cases", () => {
       // //////////
       // // MINT //
       // //////////
-      await expect(vBNX.connect(acc2Signer).mint(mintAmount))
-        .to.emit(vBNX, "Mint")
-        .withArgs(acc2, mintAmount, mintAmount, mintAmount);
+      const tx = await vBNX.connect(acc2Signer).mint(mintAmount);
+      await expect(tx).to.emit(vBNX, "Mint").withArgs(acc2, mintAmount, vTokenMintAmount, vTokenMintAmount);
       [error, balance, borrowBalance] = await vBNX.connect(acc2Signer).getAccountSnapshot(acc2);
       expect(error).to.equal(Error.NO_ERROR);
-      expect(balance).to.equal(mintAmount);
+      expect(balance).to.equal(vTokenMintAmount);
       expect(borrowBalance).to.equal(0);
       [error, liquidity, shortfall] = await Comptroller.connect(acc2Signer).getAccountLiquidity(acc2);
       expect(error).to.equal(Error.NO_ERROR);
-      expect(liquidity).to.equal(new BigNumber(mintAmount * collateralFactor).multipliedBy(vBNXPrice));
+      expect(liquidity).to.equal(new BigNumber(mintAmount).multipliedBy(collateralFactor).multipliedBy(vBNXPrice));
       expect(shortfall).to.equal(0);
       // ////////////
       // // Borrow //
       // ////////////
       // Supply WBTC to market from 2nd account
       await expect(vBSW.connect(acc1Signer).mint(mintAmount))
         .to.emit(vBSW, "Mint")
-        .withArgs(await acc1Signer.getAddress(), mintAmount, mintAmount, mintAmount);
-
+        .withArgs(await acc1Signer.getAddress(), mintAmount, vTokenMintAmount, vTokenMintAmount);
       [error, balance, borrowBalance] = await vBSW
         .connect(acc2Signer)
```

Comment:
```suggestion
        .connect(acc1Signer)
```

Reply:
We are supplying with acc2 to the market so acc1 to be able to borrow.

Reply:
This is fixed